sky/packages/updater/lib/bundle.dart

```diff
@@ -0,0 +1,64 @@
+// Copyright 2015 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:async';
+import 'dart:convert';
+import 'dart:io';
+import 'dart:typed_data';
+
+import 'package:yaml/yaml.dart' as yaml;
+
+const String kBundleMagic = '#!mojo mojo:sky_viewer\n';
+
+Future<List<int>> _readBytesWithLength(RandomAccessFile file) async {
+  var buffer = new ByteData(4);
```

Comment:
please don't use "var"

(I updated the style guide recently to mention this explicitly; see https://github.com/flutter/engine/blob/master/sky/specs/style-guide.md )